### ISSUE TITLE
Unable to determine cause of warning

### ISSUE THREAD

**Issue by tech2serve** [WordPress/jquery-migrate-helper#15]
I help maintain a Wordpress plugin. My dev site is running WordPress v5.5.3 and I have used jQuery Manager for Wordpress (v1.10.6) to disable jQuery Migrate. I do not see any errors or warnings, and the plugin seems to be working as expected. 

Yet, jQuery Migrate Helper provides these warnings when active.

<img width="883" alt="image" src="https://user-images.githubusercontent.com/31904316/97828572-17313780-1c8d-11eb-9744-ae73676bb63c.png">

These warnings aren't particularly useful to me. Our `admin.js` does contain three `getJSON` calls, but I have not found anything with them doesn't seem consistent with the jquery documentation. I dug deeper into our `ajax.php` and also didn't find anything that appeared to me to be out of order.

More information from jQuery-migrate-helper would be useful. Is there a way to increase the amount of information provided by the helper plugin?

Are the warnings being triggered by something else? 

Any pointers you can give me?

Thanks.

**Comment by tech2serve** [comment]
I'm now exploring that the warning is coming from `typeahead.js`. We have three `Bloodhound` instantiations.

**Comment by Clorith** [comment]
The problem here is that you're trying to parse JSON data, without verifying that it is a valid JSON response first, so the warning is thrown because it isn't.

Add some checks around your `jQuery.parseJSON` calls and should be good.

I'll be closing this ticket, as it's an implementation case, and not a problem with this plugin. If you need further support, I recommend the support forums for that purpose :)

**Comment by tech2serve** [comment]
I opened the Issue because there are no files in our plugin that use `parseJSON`...well, directly anyway. And that is why the "warning" message was so hard to track down. At first I thought perhaps `getJSON` used it, and that was the source of the warning. But that turned out to be a rabbit hole.

Then I started looking at `Bloodhound` in `typeahead.js`, which we do use on that page. And in exploring their source code, they use `parseJSON`.  So, I'm working to rewrite to `autocomplete`.

I left the ticket open because I didn't find the warning particularly helpful. If it is possible to provide a trace, that would have dramatically cut down the amount of time I spent searching. 

In reality, I haven't actually confirmed that I'm on the right track.
